webapp/pages/profile/_id/_slug.vue

```diff
@@ -142,69 +142,50 @@
           </ds-card>
         </ds-space>
       </ds-flex-item>
+
       <ds-flex-item :width="{ base: '100%', sm: 3, md: 5, lg: 3 }">
-        <ds-flex :width="{ base: '100%' }" gutter="small">
+        <ds-flex class="user-profile-posts-list" :width="{ base: '100%' }" gutter="small">
           <ds-flex-item class="profile-top-navigation">
             <ds-card class="ds-tab-nav">
-              <ds-flex>
-                <ds-flex-item
-                  v-tooltip="{
-                    content: $t('common.your.post', null, user.contributionsCount),
-                    placement: 'right',
-                    delay: { show: 500 },
-                  }"
-                  class="ds-tab-nav-item pointer ds-tab-nav-item-active"
-                  @click="tabActivity('posts', $event)"
-                >
-                  <ds-space margin="small">
-                    <!-- TODO: find better solution for rendering errors -->
-                    <no-ssr>
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-05T20:50:57Z" title="Wednesday, June 5th 2019, 10:50:57 pm +02:00">Jun 5, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -144,16 +144,17 @@
                     delay: { show: 500 },
                   }"
                   class="ds-tab-nav-item pointer ds-tab-nav-item-active"
-                  @click="tabActivity('posts', $event)"
                 >
-                  <ds-space margin="small">
-                    <!-- TODO: find better solution for rendering errors -->
-                    <no-ssr>
```


@ogerly could you check if `no-ssr` is really necessary here?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-06-06T03:54:54Z" title="Thursday, June 6th 2019, 5:54:54 am +02:00">Jun 6, 2019</time>_

---


> no-ssr

@roschaefer  

![FireShot Capture 133 - Human Connection - Human Connection - localhost](https://user-images.githubusercontent.com/1324583/59006911-8d293900-8824-11e9-897b-817add35d254.png)

so after examination of no-ssr.

without no-ssr the following happens: when the page is reloaded hard, i.e. "F5", the data will be 
server-side and client-side rendered. like on the picture above the value of shouted (recommendations) is there twice. this tab has NO no-ssr. the other two have no-ssr. 

so i still leave no-ssr. because i'm not quite sure. and still don't understand enough about vue. which is slowly changing. but no-ssr is a pretty useful thing. 

## Example (shouted): 
see picture above

    ` <ds-number :label="$t('profile.shouted')">
              <hc-count-to slot="count" :end-val="user.shoutedCount" />
       </ds-number>`

 
but this is only possible with a hard reload (F5) 

So I take this variant.

`<no-ssr placeholder="Loading...">
                        <ds-number :label="$t('profile.shouted')">
                          <hc-count-to slot="count" :end-val="user.shoutedCount" />
                        </ds-number>
                      </no-ssr>`